models/boleto.go

```diff
@@ -69,6 +69,7 @@ type BoletoView struct {
 // NewBoletoView cria um novo objeto view de boleto a partir de um boleto request, codigo de barras e linha digitavel
 func NewBoletoView(boleto BoletoRequest, response BoletoResponse, bankName string) BoletoView {
 	boleto.Authentication = Authentication{}
+	boleto.Title.OrderDiscountsByDate()
```

Comment:
```suggestion
	boleto.Title.SortDiscountsByDate()
```
Na literatura de coisas de computação, geralmente, a galera utiliza sort pra indicar a ordenação de algo.